chapters/fragops.txt

```diff
@@ -955,22 +956,24 @@ fragments generated by <<primsrast-polygons-basic,back-facing>>
 fragments generated by <<primsrast-polygons-basic,front-facing polygons>> or
 any other primitives.
 
-The comparison performed is based on the elink:VkCompareOp, compare mask
-[eq]#s~c~# , and stencil reference value [eq]#s~r~# of the relevant state
-set.
-The compare mask and stencil reference value are set by either the
+The comparison operation performed is determined by a elink:VkCompareOp
+value set by
+ifdef::VK_VERSION_1_3,VK_EXT_extended_dynamic_state[]
+flink:vkCmdSetDepthCompareOp::pname:depthCompareOp, or by
```

Comment:
I don't think I've seen this "flink:A::pname:B" usage before (only with slink). But if it renders something reasonable, then no objection.

Reply:
It's used in a couple of dozen places, but not often.